src/main/java/com/google/sps/servlets/utils/VideoInformation.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets.utils;
+
+import com.google.api.services.youtube.model.Video;
+import com.google.api.services.youtube.model.VideoListResponse;
+import com.google.api.services.youtube.model.VideoSnippet;
+import com.google.api.services.youtube.model.VideoStatistics;
+
+public class VideoInformation {
+  String videoName;
+  String videoAuthor;
+  int numLikes;
+  int numDislikes;
+  String publishDate;
+
+  public VideoInformation(VideoListResponse videoResponse) {
+    Video video = videoResponse.getItems().get(0);
```

Comment:
To ensure that the Youtube API is returning what we expect it's useful to verify that they are only returning us exactly one Video object.

We can do this compactly with `Iterables.getOnlyElement` from the Guava library (https://guava.dev/releases/20.0/api/docs/com/google/common/collect/Iterables.html) which will throw an exception if there is not exactly 1 element in the supplied list.

You can also write a method to do this manually if you don't want to bother with adding a new maven dependency.